src/main/common/pwl.h

```diff
@@ -25,7 +25,6 @@
 
 
 #define PWL_DECLARE(name, size, xMinV, xMaxV)                           \
-    STATIC_ASSERT((xMinV) < (xMaxV), "xMinV must be less than xMaxV");  \
```

Comment:
Float is not the problem and PWL_DECLARE is not used in this PR. Can you elaborate on the problem you encountered?

Reply:
@ledvinap so if i do this in our C code, it fails:
`STATIC_ASSERT(2.0f > 1.0f, "test");`
`error: expression in static assertion is not an integer constant expression [-Werror=pedantic]`
it only works with integers like that:
`STATIC_ASSERT(2 > 1, "test");`

i know i don't use `PWL_DECLARE` in this PR, but just for future use.

Reply:
That's the  `-pedantic` nonsense. Best CoA is to remove it from Makefile